packages/flutter_tools/test/src/fake_vm_services.dart

```diff
@@ -26,14 +26,14 @@ class FakeVmServiceHost {
     ), httpAddress: httpAddress, wsAddress: wsAddress);
     _applyStreamListen();
     _output.stream.listen((String data) {
-      final Map<String, Object> request = json.decode(data) as Map<String, Object>;
+      final Map<String, Object?> request = json.decode(data) as Map<String, Object?>;
```

Comment:
`exceptionPauseMode` is nullable, this needs to handle null values:
https://github.com/flutter/flutter/blob/e7b7ebc066c1b2a5aa5c19f8961307427e0142a6/packages/flutter_tools/test/general.shard/tracing_test.dart#L26